src/decode.h

```diff
@@ -156,7 +156,12 @@
         _(cjalr, 1)                        \
         _(cadd, 0)                         \
         _(cswsp, 0)                        \
-    )
+    )                                      \
+    _(fuse1, 0)                            \
```

Comment:
Add a comment,
> macro operation fusion: convert specific RISC-V instruction patterns into shorter and equivalent code

Reply:
In JVM, such technique is called "quickening optimization."
See https://www.complang.tuwien.ac.at/andi/papers/dotnet_06.pdf